src/components/side_panel/components/panel_color_picker/panel_color_picker.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, useExternalListener, useRef, useState } from "@odoo/owl";
+import { Rect, SpreadsheetChildEnv } from "../../../../types";
+import { ColorPicker } from "../../../color_picker/color_picker";
+import { ColorPickerWidget } from "../../../color_picker/color_picker_widget";
+import { css, cssPropertiesToCss } from "../../../helpers";
+import { getBoundingRectAsPOJO } from "../../../helpers/dom_helpers";
+import { Section } from "../section/section";
+
+interface State {
+  pickerOpened: boolean;
+}
+
+interface Props {
+  currentColor?: string;
+  onColorPicked: (color: string) => void;
+}
+
+const TRANSPARENT_BACKGROUND_SVG = /*xml*/ `
+<svg xmlns="http://www.w3.org/2000/svg" width="10" height="10">
+  <path fill="#d9d9d9" d="M5 5h5v5H5zH0V0h5"/>
+</svg>
+`;
+
+css/* scss */ `
+  .o-panel-color-picker-button {
+    width: 15px;
+    height: 15px;
+    cursor: pointer;
+    border: 1px solid #aaa;
+    background-position: 1px 1px;
+    background-image: url("data:image/svg+xml,${encodeURIComponent(TRANSPARENT_BACKGROUND_SVG)}");
+  }
+`;
+
+export class PanelColorPicker extends Component<Props, SpreadsheetChildEnv> {
```

Comment:
I'd name this component differently. It has nothing specific to side panels anymore and could be used anywhere. Maybe `RoundColorPicker` or any other name ?